### PR TITLE
Train model v13

### PR DESCRIPTION
**Pull request by MortenTabaka**

### CODE REVIEW

**Comment by MortenTabaka** [comment]
Notebook:
https://drive.google.com/file/d/1_MreOQti76DU_BZ_2S5at67TpFZlEj3U/view?usp=sharing


`test_mIoU` reached is 0.572

No weights or learning rate decay was used. Therefore, it possibly is reason for quite good results in v12 (https://github.com/MortenTabaka/Semantic-segmentation-of-LandCover.ai-dataset/pull/26#issuecomment-1502872167)

### deeplabv3plus_v13.1.1

```
  model_compile_parameters:
    optimizer: <tensorflow.python.keras.optimizer_v2.adam.Adam object at 0x7fc548185dc0>
    loss_function:
      object: <bound method SemanticSegmentationLoss.basnet_hybrid_loss of <src.features.loss_functions.SemanticSegmentationLoss
        object at 0x7fc548185c10>>
      initial_learning_rate: 0.001
      final_learning_rate: null
```

### deeplabv3plus_v13.1.2
```
  model_compile_parameters:
    optimizer: <tensorflow.python.keras.optimizer_v2.adam.Adam object at 0x7fc4744e68b0>
    loss_function:
      object: <bound method SemanticSegmentationLoss.basnet_hybrid_loss of <src.features.loss_functions.SemanticSegmentationLoss
        object at 0x7fc548185c10>>
      initial_learning_rate: 0.0001
      final_learning_rate: null
```
